deploy/helm/kuberhealthy/values.yaml

```diff
@@ -192,6 +192,7 @@ check:
       registry: kuberhealthy
       repository: pod-restarts-check
       tag: v2.3.3
```

Comment:
Same as above, I'm not sure if I need to do anything with this tag?  As this needs to points to a version that hasn't been released yet so maybe the release pipeline updates this?

Reply:
I believe this version `v2.3.3` has been released? https://hub.docker.com/repository/registry-1.docker.io/kuberhealthy/pod-restarts-check/tags?page=1&ordering=last_updated

hmm we don't have the best release process for specific versions for these external checks -- when a pr for external checks get merged in we do have a github action in place for build and pushing "latest". For specific versions, we build and push manually :/

Reply:
ok cool - makes sense, so is it best to change that to latest in this PR and let the followup PR set the correct tag?

Reply:
Yep... this is the flow I would expect:

- third party check makes a new release version
- third party check opens a PR to edit this values.yaml file for the helm chart
- helm chart static files are regenerated automatically